spec/apps/hanami/doc/openapi.yaml

```diff
@@ -212,16 +236,11 @@ paths:
               example:
                 items:
                 - secrets
-        '401':
```

Comment:
Hmm, why `401` responses are gone?
It looks like a regression.

Reply:
I have researched this issue, missed it unfortunately earlier. I think it's not a regression, it's just that this section of documentation in rails is manually added, maybe it's part of testing that checks that manually added documentation won't be removed automatically.

When I generated the hanami documentation I deleted the whole file and re-generated it, I will add this part manually if I understand the reason for the existence of this part correctly.